model_analyzer/config/input/config_command_profile.py

```diff
@@ -920,14 +920,15 @@ def _add_run_search_configs(self):
             ConfigField(
                 "run_config_search_mode",
                 flags=["--run-config-search-mode"],
-                choices=["brute", "quick"],
+                choices=["brute", "quick", "optuna"],
                 field_type=ConfigPrimitive(str),
                 default_value=DEFAULT_RUN_CONFIG_SEARCH_MODE,
                 description="The search mode for Model Analyzer to find and evaluate"
                 " model configurations. 'brute' will brute force all combinations of"
                 " configuration options.  'quick' will attempt to find a near-optimal"
                 " configuration as fast as possible, but isn't guaranteed to find the"
-                " best.",
+                " best. 'optuna' is a more generalized version of quick search allowing "
+                " the user to quick search over any parameter.",
```

Comment:
qq: is optuna using quick search under the hood? The description sounds like it is some form of our quick search algorithm.

Reply:
No, it is not. It is using their own algorithm to search the space.

Reply:
Then I would remove "quick search" in the description to avoid confusion:
```suggestion
                " best. 'optuna' is a more generalized search algorithm allowing "
                " the user to quickly search over any parameter.",
```

Reply:
Fixed